### PR TITLE
feat(add-license)

### PR DESCRIPTION
**Pull request by ecovirtual**
Changes:

Update Package.json with extra MIT license field
Update Contributing file with extra description for branch naming

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "se-2",
+  "name": "se-2-noir",
   "version": "0.0.1",
```
Comment (FilipHarald) [inline package.json:3]: maybe we should change this to `1.0.0`? :)

**Review by FilipHarald — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (ecovirtual) [reply package.json:3]: Good point.... mmm I think this should be 0.1.0 actually

Comment (FilipHarald) [reply package.json:3]: OK, sure.

Comment (ecovirtual) [reply package.json]: Done

**Pull request merged**